server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/default.ftl

```diff
@@ -45,6 +45,7 @@
       <script src="/resources/external/flot/jquery.flot.time.js"></script>
       <script src="/resources/external/flot/jquery.flot.resize.js"></script>
       <link rel="stylesheet" href="/resources/external/bootstrap/css/bootstrap.css" />
+      <link rel="stylesheet" href="/resources/external/bootstrap/css/bootstrap-icons.css" />
```

Comment:
This new dependency (bootstrap icons are a separate dependency than the bootstrap bundle) requires an update to the LICENSE files.

From the top of the css file, I see this info to use:
```
/*!
 * Bootstrap Icons v1.11.1 (https://icons.getbootstrap.com/)
 * Copyright 2019-2023 The Bootstrap Authors
 * Licensed under MIT (https://github.com/twbs/icons/blob/main/LICENSE)
 */
```

Related: I noticed flot and jquery have license info in all three of:

* LICENSE
* server/monitor/src/main/appended-resources/META-INF/LICENSE
* assemble/src/main/resources/LICENSE

But bootstrap and datatables are only in the first two, but should probably be in the third as well.

The first one is for the source tarball. The last one is for the binary tarball, and the middle one is for the monitor's jar, which is also contained in the binary tarball, but should still list things at the root of the tarball.

Reply:
Should be fixed in 80a743c